contentcuration/contentcuration/frontend/channelEdit/views/TreeView/TreeViewBase.vue

```diff
@@ -235,12 +235,21 @@
     <PublishSidePanel
       v-if="showPublishSidePanel"
       @close="showPublishSidePanel = false"
+      @published="handleChannelPublished"
```

Comment:
Seems like `handleChannelPublished` isn't defined on the component. And I think it's because we don't need to do anything when the `published` event is emitted. So we can perhaps also remove the event emit within the `PublishSidePanel` component?